features/comment.feature

```diff
@@ -2,7 +2,8 @@ Feature: Manage WordPress comments through the REST API
 
   Background:
     Given a WP install
-    And I run `wp plugin install rest-api --activate`
+    When I run `wp core version`
+	Then STDOUT should be a version string >= 4.7
```

Comment:
Of course. The pull request is for your benefit and those that use restful at large. I will continue to use my restful fork until some future where you reach my functionality level or we merge. I have a well working fork and my needs as such that I won't wait or lessen functionality. That's one of the benefits of forks. :-)